src/async_fuse/memfs/kv_engine/mod.rs

```diff
@@ -225,8 +232,24 @@ pub trait KVEngine: Send + Sync + Debug + Sized {
     /// Lease grant
     async fn lease_grant(&self, ttl: i64) -> DatenLordResult<i64>;
```

Comment:
lease_grant() can be removed because we have abstract Session.  This function is not used in the code.

Reply:
Ok, now it is removed.